### DISCUSSION TITLE
MPD and windows

### DISCUSSION THREAD

**Discussion by davidremmons** [MusicPlayerDaemon/MPD#1572] [General]
I am trying to find a copy of MPD that will run on a Windows2000 server. Does it exist? I notice that all the Windows links by third party sites bring you to the client side only and state that the file in the link is not found. Where are the source files for WPD win32? Any help would be appreciated. Thank you.

**Comment by MaxKellermann** [comment]
MPD is not compatible with Windows 2000. It uses APIs that are only available since Windows Vista.

**Reply by davidremmons** [reply]
Thank you for your reply. I have it working now on a windows 64bit MPD.exe yet I have some legacy machines that are Windows 7 32bit and would like to know if there are any 32bit versions working of MPD.exe? On the website win32 directory is full of versions but none are actual 32bit compiled executables. If they are 64bit, why are they in win32 directories? I don't quite understand? And where are the binaries for these versions? Quite a few websites that offer help with running MPD have links that no longer work.  Appreciate your assistance.

**Reply by MaxKellermann** [reply]
WIN32 is a misleading name coined by Microsoft, it applies to both Windows for IA32 and Windows for AMD64.
There are no precompiled MPD executables for 32 bit Windows.
